tests/integration/usage_statistics/test_usage_stats_common_messages_are_sent_v3api.py

```diff
@@ -94,7 +94,7 @@ def test_common_usage_stats_are_sent_no_mocking(
     context._usage_statistics_handler._close_worker()
 
     # Make sure usage stats are enabled
-    assert not context._check_global_usage_statistics_opt_out()
+    assert context._get_global_usage_statistics_override()
```

Comment:
I know we had this issue before but I don't love directly accessing a private method here - is it possible to get around this at all? Do we need this assertion?

Feels like we might be testing an implementation detail

Reply:
I can understand this concern. I think we can get around this by checking the boolean flag on `anonymous_usage_statistics` property, which we do directly below. If that is the case then we can just remove this line